docs/standard/base-types/how-to-display-milliseconds-in-date-and-time-values.md

```diff
@@ -13,40 +14,40 @@ helpviewer_keywords:
   - "milliseconds [.NET]"
 ms.assetid: ae1a0610-90b9-4877-8eb6-4e30bc5e00cf
 ---
-# How to: Display Milliseconds in Date and Time Values
+# How to: Display milliseconds in date and time values
 
-The default date and time formatting methods, such as <xref:System.DateTime.ToString?displayProperty=nameWithType>, include the hours, minutes, and seconds of a time value but exclude its milliseconds component. This article shows how to include a date and time's millisecond component in formatted date and time strings.
+The default date and time formatting methods such as <xref:System.DateTime.ToString?displayProperty=nameWithType>, include the hours, minutes, and seconds of a time value but exclude its milliseconds component. This article shows how to include a date and time's millisecond component in formatted date and time strings.
```

Comment:
```suggestion
The default date and time formatting methods, such as <xref:System.DateTime.ToString?displayProperty=nameWithType>, include the hours, minutes, and seconds of a time value but exclude its milliseconds component. This article shows how to include a date and time's millisecond component in formatted date and time strings.
```

Reply:
Such as isn't restrictive in this context.

Reply:
Ok, updated the content.